parcels/grid.py

```diff
@@ -78,7 +79,16 @@ def from_netcdf(cls, filenames, variables, dimensions, **kwargs):
             fields[var] = Field.from_netcdf(var, dimensions, dsets, **kwargs)
         u = fields.pop('U')
         v = fields.pop('V')
-        return cls(u, v, u.depth, u.time, fields=fields)
+
+        # assign time_origin if the time dimension has units and calendar
+        try:
+            time_units = dsets[0][dimensions['time']].units
+            calendar = dsets[0][dimensions['time']].calendar
+            time_origin = num2date(0, time_units, calendar)
```

Comment:
Actually I just manually rescaled the time data in the original netCDF. The time was been days since an origin for a particular, whereas other fields were in seconds since the same origin. The interpolator appeared to just be using the final time-step of the incorrectly scaled field for the entire execution loop, and so initially I did not pick up that there was a problem.

I suggest that fields have their own `time`, but that a warning is displayed when values are interpolated outside of that time range for any field? This check could be added in `eval` function of the `Field` class?